website/src/data/components/marimekko/props.js

```diff
@@ -0,0 +1,377 @@
+    },
+    {
+        key: 'margin',
+        help: 'Chart margin.',
+        type: 'object',
+        required: false,
+        controlType: 'margin',
+        group: 'Base',
+    },
+    themeProperty,
+    {
+        key: 'colors',
+        help: 'Defines color range.',
+        type: 'string | Function | string[]',
+        required: false,
+        defaultValue: defaults.colors,
+        controlType: 'ordinalColors',
+        group: 'Style',
+    },
+    ...defsProperties('Style', ['svg', 'api']),
+    {
+        key: 'showcase pattern usage',
+        flavors: ['svg'],
+        help: 'Patterns.',
+        description: `
+            You can use \`defs\` and \`fill\` properties
+            to use patterns, see
+            [dedicated guide](self:/guides/patterns)
+            for further information.
+        `,
+        type: 'boolean',
+        controlType: 'switch',
+        group: 'Style',
+    },
+    {
+        key: 'borderWidth',
+        help: 'Slices border width.',
+        type: 'number',
+        required: false,
+        defaultValue: defaults.borderWidth,
+        controlType: 'lineWidth',
+        group: 'Style',
+    },
+    {
+        key: 'borderColor',
+        help: 'Method to compute border color.',
+        type: 'string | object | Function',
+        required: false,
+        defaultValue: defaults.borderColor,
+        controlType: 'inheritedColor',
+        group: 'Style',
+    },
+    ...axesProperties(),
+    {
+        key: 'enableGridX',
+        help: 'Enable/disable x grid.',
+        type: 'boolean',
+        required: false,
+        defaultValue: defaults.enableGridX,
+        controlType: 'switch',
+        group: 'Grid & Axes',
+    },
+    {
+        key: 'gridXValues',
+        group: 'Grid & Axes',
+        help: 'Specify values to use for vertical grid lines.',
+        type: 'number[]',
+        required: false,
+    },
+    {
+        key: 'enableGridY',
+        help: 'Enable/disable y grid.',
+        type: 'boolean',
+        required: false,
+        defaultValue: defaults.enableGridY,
+        controlType: 'switch',
+        group: 'Grid & Axes',
+    },
+    {
+        key: 'gridYValues',
+        group: 'Grid & Axes',
+        help: 'Specify values to use for horizontal grid lines.',
+        type: 'number[]',
+        required: false,
+    },
+    {
+        key: 'layers',
+        group: 'Customization',
+        help: 'Defines the order of layers and add custom layers.',
+        description: `
+            You can also use this to insert extra layers
+            to the chart, the extra layer must be a component.
+            
+            The layer component which will receive the chart's
+            context & computed data and must return a valid SVG element
+            for the \`Marimekko\` component.
+            
+            The props passed to layers have the following structure:
+            
+            \`\`\`
+            {
+                data: ComputedData<RawDatum>[],
```

Comment:
I know there are a few extra props on the layer context, did you want to add them here?

Reply:
Yes, good catch!

Reply:
Added.